controllers/drplacementcontrol_controller.go

```diff
@@ -2643,3 +2668,147 @@ func constructVRGFromView(viewVRG *rmn.VolumeReplicationGroup) *rmn.VolumeReplic
 
 	return vrg
 }
+
+func drpcInAdminNamespace(drpc *rmn.DRPlacementControl, ramenConfig *rmn.RamenConfig) bool {
```

Comment:
(nit) function order in our code is usually caller first and definition to follow caller. IOW we should have had `ensureDRPCValidNamespace` before `drpcInAdminNamespace` and so on ...

Reply:
done